Misc/NEWS.d/next/Tests/2020-03-01-13-49-52.bpo-12915.pQsDV1.rst

```diff
@@ -0,0 +1,3 @@
+When filesystem encoding cannot encode the Unicode string used for a filename
+continue testing with the next example.
+Patch by M Felt.
```

Comment:
I don't think that it's worth it to add a NEWS entry for this change. Please remove this file.

Reply:
needs a flag I expect now, but done.